offchain-modules/packages/scripts/src/cardano-integ.ts

```diff
@@ -1,18 +1,19 @@
-import fs from 'fs';
+// import fs from 'fs';
 import path from 'path';
+// eslist-disable-next-line
 import * as CardanoWasm from '@emurgo/cardano-serialization-lib-nodejs';
-import { ValInfos } from '@force-bridge/cli/src/changeVal';
+// import { ValInfos } from '@force-bridge/cli/src/changeVal';
 import { KeyStore } from '@force-bridge/keystore/dist';
 import { OwnerCellConfig } from '@force-bridge/x/dist/ckb/tx-helper/deploy';
-import { AdaConfig, Config, WhiteListEthAsset, CkbDeps } from '@force-bridge/x/dist/config';
+import { Config, CkbDeps } from '@force-bridge/x/dist/config';
 import { asyncSleep, privateKeyToCkbPubkeyHash, writeJsonToFile, genRandomHex } from '@force-bridge/x/dist/utils';
 import { logger, initLog } from '@force-bridge/x/dist/utils/logger';
 import * as utils from '@force-bridge/x/dist/xchain/ada/utils';
-import { AdaChain } from '@force-bridge/x/dist/xchain/ada/wallet-interface';
+// import { AdaChain } from '@force-bridge/x/dist/xchain/ada/wallet-interface';
```

Comment:
If these libraries aren't used can we remove the lines instead of merely commenting them out?